[Fix #599] Add Python documentation (#600)
apache/incubator-kie-kogito-docs: serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

include::../../pages/_common-content/camel-valid-responses.adoc[]

[[con-func-python]
== Python custom function

> {product_name} provides an implementation of a custom function to execute embedded Python scripts and functions. See xref:use-cases/advanced-developer-use-cases/integrations/custom-functions-knative.adoc[Invoking Python from {product_name}]

Reviewer: ```suggestion
{product_name} implements a custom function to execute embedded Python scripts and functions. See xref:use-cases/advanced-developer-use-cases/integrations/custom-functions-knative.adoc[Invoking Python from {product_name}]
```
